ui/src/reducers/beamline.js

```diff
@@ -26,6 +26,7 @@ export const INITIAL_STATE = {
       state: 'undefined',
       msg: 'UNKNOWN',
       readonly: false,
+      invert_bg_color: false,
```

Comment:
I think the new property ends up in the `uiproperties` state, not in `beamline.hardwareObjects`. That being said, I don't think you need initial values at all, so you can just remove the changes in the file.